tools/travis/lnx-cibuild.sh

```diff
@@ -63,6 +63,7 @@ export PATH=${SOURCE_DIRECTORY}/_thirdparty/Comet:$PATH
 export PATH=${SOURCE_DIRECTORY}/_thirdparty/Sirius:$PATH
 export PATH=${SOURCE_DIRECTORY}/_thirdparty/SpectraST:$PATH
 export PATH=${SOURCE_DIRECTORY}/_thirdparty/Percolator:$PATH
+export PATH=${SOURCE_DIRECTORY}/_thirdparty/MaRaClusterAdapter:$PATH
```

Comment:
You need to add the Thirdparty folder to the path. I.e. MaRaCluster instead of MaRaClusterAdapter